src/communication/local_comm_system.py

```diff
@@ -0,0 +1,62 @@
+import queue
+from threading import Lock, Thread
+import time
+from src.communication.messages import Message, MessageType
+
+
```

Comment:
If you're going to use comments for once, put something in here about this being a base class that each subsystem will create their own comms classes from. Really only just some context for what this does.